pkg/networkpath/traceroute/winconn/winconn_windows.go

```diff
@@ -24,6 +24,15 @@ var (
 )
 
 type (
+	// RawConnWrapper is an interface that abstracts the raw socket
+	// connection for Windows
+	RawConnWrapper interface {
```

Comment:
Do you want to also move this to `tcpv4_windows.go` perhaps to mimic `rawConnWrapper`? Also wondering if we can use one interface between the unix and windows code.

Reply:
That's a good catch, this should actually be used in `udpv4_windows.go` as well. Just changed it.

I think we can, but it's going to take a bit of re-working. The Windows one has `ListenPackets` as a function, the Unix one directly wraps the `ipv4.RawConn` functions. So a bunch of tests would need to be re-written again to make that work. I can try to do that as a follow up and I think it might make sense to make the unix interface more similar to what's been done for Windows.

Reply:
Yes, that would be great as a follow on.